config/ocp-observatorium-test.yaml

```diff
@@ -29,107 +29,186 @@ loki:
 
 scenarios:
   highVolumeReads:
-    enabled: true
+    enabled: false
```

Comment:
maybe need to move back to "True" :-)

Reply:
Yep. This is still a draft so some things are still in progress.